tools/examples/src/main.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * @license
+ * Copyright 2019 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { lstatSync, readdirSync } from 'fs';
+import { join } from 'path';
+import { bold, green } from 'chalk';
+import {
+  getExamplePackageMetadata,
+  ExamplePackageMetadata,
+  ExampleMetadata,
+} from './metadata';
+import { generateExamplesLibBarrelFile } from './generate-examples-lib-barrel';
+import { generateDemosAppExamplesModule } from './generate-examples-module';
+import { generateDemosAppRoutingModule } from './generate-routing-module';
+import { generateDemosAppNavItems } from './generate-nav-items';
+
+export const EXAMPLES_ROOT = join(
```

Comment:
don't use __dirname in case this would be different in the dist folder can you use process.argv to provide the paths via cli?

Reply:
It's the same depth as in dist. If we move it around we can adjust later